### PR TITLE
Deprecate Ordering for Iterable

### PR DESCRIPTION
**Pull request by NthPortal**

### CODE REVIEW

File: test/files/neg/iterable-ordering.check [hunk 1]
```diff
@@ -0,0 +1,6 @@
+iterable-ordering.scala:2: warning: method Iterable in object Ordering is deprecated (since 2.13.0): Iterables are not guaranteed to have a consistent order; if using a type with aconsistent order (e.g. Seq), use its Ordering (found in the Ordering.Implicits object)
```
Comment (Ichoran) [inline test/files/neg/iterable-ordering.check]: Missing a space in `aconsistent`.

**Review by Ichoran — APPROVED** [review]
Good idea.  Looks fine to me!  All I found was one typo.

File: test/files/neg/iterable-ordering.check (hunk 1 above)
Comment (NthPortal) [reply test/files/neg/iterable-ordering.check]: thanks, good catch!

**Review by lrytz — APPROVED** [review]
Thanks!

**Pull request merged**